package.json

```diff
@@ -8,14 +8,8 @@
     "url": "git@github.com:eyespot-agency/eyespot-starter-gatsby.git"
   },
   "author": "EMIKETIC Technologies <contact@emiketic.com>",
-  "license": "MIT",
-  "homepage": "https://www.eyespot.agency",
-  "keywords": [
-    "gatsby",
-    "react",
-    "gatsby starter",
-    "eyespot"
-  ],
+  "license": "Proprietary",
+  "homepage": "https://www.emiketic.com",
```

Comment:
@raed-emiketic You need to switch back to the initial licence and meta data that were removed above. Starter kit is under MIT

Reply:
OKay I'll fix it